README.md

```diff
@@ -138,69 +137,128 @@ You can take a look at the tasks you created using the [Tekton CLI](https://gith
 ```
 $ tkn task ls
 
-NAME               AGE
-openshift-client   58 seconds ago
-s2i-java-8         1 minute ago
+NAME                AGE
+apply-manifests     10 seconds ago
+buildah             1 minute ago
+s2i-python-3        50 seconds ago
+update-deployment   4 seconds ago
 ```
 
 ## Create Pipeline
 
 A pipeline defines a number of tasks that should be executed and how they interact with each other via their inputs and outputs.
 
-In this tutorial, you will create a pipeline that takes the source code of the PetClinic application from GitHub and then builds and deploys it on OpenShift using [Source-to-Image (S2I)](https://docs.openshift.com/container-platform/4.1/builds/understanding-image-builds.html#build-strategy-s2i_understanding-image-builds).
+In this tutorial, you will create a pipeline that takes the source code of the application from GitHub and then builds and deploys it on OpenShift.
 
-<p align="center"><img src="images/pipeline-diagram.svg" width="700" alt="Pipeline Diagram" /></div>
+![Pipeline Diagram](images/pipeline-diagram.png)
```

Comment:
Diagram doesn't match the screenshot of the pipeline. In one `build-ui` and `build-api` run in parallel, in the other one they are sequential.

Reply:
There is a bug in Devconsole diagram

Reply:
Even in the latest dev console, it is not showing properly

![bug-flow](https://user-images.githubusercontent.com/19270240/69866919-63306b00-12cb-11ea-9222-6a8b6f284856.png)